src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/PackageLibs.targets

```diff
@@ -140,6 +140,12 @@
       <FilesToPackage Condition="'%(FilesToPackage.SubFolder)' != ''">
         <TargetPath>%(TargetPath)%(FilesToPackage.SubFolder)</TargetPath>
       </FilesToPackage>
+
+      <!-- Include source files for symbol packages. -->
+      <FilesToPackage Include="@(Compile->'%(FullPath)')" Condition="$([System.String]::new('%(FullPath)').StartsWith('$(ProjectDir)'))">
```

Comment:
What about the source code not under the project directory? Is the expectation that the source code will be reference only and not build able?